kontrakter/politi/innsettelsesordre/1.0/innsettelsesordre.schema.json

```diff
@@ -501,10 +427,7 @@
         "besluttetAv": {
           "$ref": "#/definitions/ansattPerson"
         },
-        "tidspunkt": {
-          "type": "string",
-          "format": "date-time"
-        },
+        "tidspunkt": { "$ref": "#/definitions/datoTid" },
```

Comment:
Tidspunkt er litt generisk. Er dette tidspunktet ordren ble undertegnet? Eller er det starttidspunkt for fengslingen?

Reply:
Enig i at vi burde hatt en mer presis formulering. Vi forstår dette som tidspunktet for beslutningen. (Dere har jo lagt på eget felt for fra- og tildato for selve fengslingen nå også.)

Reply:
Da endrer jeg til beslutningsTidspunkt.